ggml/src/ggml-cuda/ggml-cuda.cu

```diff
@@ -80,6 +80,12 @@
 #include <string>
 #include <vector>
 
+
+
+
+void ggml_cuda_op_hardswish(ggml_backend_cuda_context & ctx, ggml_tensor * dst);
```

Comment:
Look at how things are done in this repo: we don't define prototypes here, but in an op-specific file.